AK/JsonValue.h

```diff
@@ -102,6 +102,13 @@ class JsonValue {
     u32 to_u32(u32 default_value = 0) const { return to_number<u32>(default_value); }
     u64 to_u64(u64 default_value = 0) const { return to_number<u64>(default_value); }
 
+#if !defined(KERNEL)
```

Comment:
`#ifndef`

Reply:
IDK, the codebase uses `defined()` about as much as `ifdef/ifndef`. I don't think it's a big enough deal to require changing it.